tasks/yesno.py

```diff
@@ -0,0 +1,301 @@
+
```

Comment:
Interesting.  I have expected that you'd just modify the hypev task appropriately.  Is there a reason to keep the old hypev task in addition to the yesno task?  Is "yesno" a fitting name?  What about the old hypev argus data?